src/index.css

```diff
@@ -0,0 +1,2 @@
+@import "tailwindcss";
+@plugin "daisyui";
```

Comment:
you are already importing tailwind and daisyui in `App.css` -  consider removing them from here!